src/components/Tabs.react.js

```diff
@@ -0,0 +1,258 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const greyColor = '#d6d6d6';
+const offWhiteColor = '#f9f9f9';
+const primaryColor = '#1975FA';
+
+const EnhancedTab = ({
+  index,
+  id,
+  label,
+  selected,
+  className,
+  style,
+  selectedClassName,
+  selected_style,
+  selectHandler,
+  value,
+  mobile_breakpoint
+}) => {
+  return (
+    <div
+      className={
+        selected
+          ? `${className || ''} ${selectedClassName || ''} tab tab--selected`
+          : `${className || ''} tab`
+      }
+      id={id}
+      style={selected ? { ...style, ...selected_style } : style}
+      onClick={() => {
+        selectHandler(index, value);
+      }}
+    >
+      <span>{label}</span>
+      <style jsx>{`
+        .tab {
+          display: inline-block;
+          background-color: white;
+          border: 1px solid ${greyColor};
+          padding: 20px;
```

Comment:
Actually, for the default tab styles in general, I prefer something like this:
```
    'width': 'calc(100% / 3)',
    'textAlign': 'center',
    'paddingTop': 15,
    'paddingBottom': 15,
    'paddingLeft': 25,
    'paddingRight': 25,
```
Full width, slightly wider than taller, centered text. Here's what it looks like:
![image](https://user-images.githubusercontent.com/1280389/42522895-450ff418-843a-11e8-94e7-c738a4f4da45.png)

Most users aren't going to change the defaults, so I want to make sure that they look as fresh as possible 😸

Reply:
I think that getting the tabs to share the entire width is especially important since tabs can contain text with varying lengths. For example:
![image](https://user-images.githubusercontent.com/1280389/42523274-4e0e958c-843b-11e8-9299-ae29975ebd07.png)

These tabs end up having different widths which looks unconventional. However, if the share an equal amount of space it ends up looking good:
![image](https://user-images.githubusercontent.com/1280389/42523334-729f55da-843b-11e8-81a1-446909a66e18.png)

Reply:
When we make the tabs equal width, we might have to do something about getting them to be the same height if the text in one tab overflows to multiple lines:
![image](https://user-images.githubusercontent.com/1280389/42523439-b8c5d78c-843b-11e8-971a-9cc570435105.png)

I'm actually not sure how we could solve that one. Perhaps we just provide a solution in our documentation to override the tab styles. It's not immediately apparent to me what that solution would be. I tried setting `height: 80px` but ended up with this:
![image](https://user-images.githubusercontent.com/1280389/42523699-7d774e80-843c-11e8-8bb6-3cc0d366c7a6.png)

Reply:
What happens if you have more than 3 tabs? Or less than? :) 
![screen shot 2018-07-11 at 10 45 09](https://user-images.githubusercontent.com/6509622/42560690-90b4b152-84f7-11e8-8496-ca30652eda37.png)
The calc(100%/3) obviously only works when there's 3 tabs, I could try it with a variable for the amount of tabs. I don't think the full width tabs look great though - they're much too wide, and too flat, like they're stretched out. I guess my solution for the multiline problem was an easy one - just have them have dynamic widths. But you're right, that is maybe a bit unconventional, and could look very silly!

Reply:
A variable actually works pretty well. Also, if you're like me and don't like the tabs to be so stretched out, you can easily set  `width: 50%` on the `container_style`! 
![screen shot 2018-07-11 at 11 02 57](https://user-images.githubusercontent.com/6509622/42561809-60dca45a-84fa-11e8-9195-88d5fa531037.png)